CONTRIBUTING.md

```diff
@@ -65,6 +65,20 @@ the API interactions should be well understood and there should be no
 technical surprises; we expect the only discussion necessary on PRs to be
 for wordsmithing and formatting.
 
+## Github Labels
+
+There is a label for each phase of the SDLC:
+
+- `reviewing proposal`: The API working group is reviewing a proposed change
+to the API.
+- `proposal finalized`: (??)
+- `validating through implementation`: One or more platforms that integrate
+   with the API implement the proposed change to provide feedback on the design
+- `validated by implementation`: (??)
```

Comment:
I'd be up for removing all of these labels (`validated by implementation`, `pr submitted`, `release candidate`, `included in release`). Once it is validated, I expect we will immediately add it as an **alpha feature** and so it will be live in master in the relevant spec file (alpha features) and we can track it there from then on.